File: python/2124.py
class Solution:
    def checkString(self, s: str) -> bool:
        existB = False
        for i in range(len(s)):
            if s[i] == 'b':
                existB = True
            elif existB:
                return False
        return True
    
if __name__ == '__main__':
    s = Solution()
    print(s.checkString(input()))

File: cpp/2124.cpp
#include<iostream>
#include<string>

using namespace std;

class Solution {
public:
    bool checkString(string s) {
        bool existB = false;
        for(int i=0; i<s.size(); i++){
            if(s[i] == 'b') existB = true;
            else if(existB) return false;
        }
        return true;
    }
};

int main(){
    Solution s;
    string str;
    cin >> str;
    cout << s.checkString(str) << endl;
    return 0;
}